content/en/tracing/setup_overview/setup/java.md

```diff
@@ -327,8 +327,23 @@ Timeout in seconds for network interactions with the Datadog Agent.
 `dd.trace.header.tags`
 : **Environment Variable**: `DD_TRACE_HEADER_TAGS`<br>
 **Default**: `null`<br>
-**Example**: `CASE-insensitive-Header:my-tag-name,User-ID:userId`<br>
-A map of header keys to tag names. Automatically apply header values as tags on traces.
+**Example**: `CASE-insensitive-Header:my-tag-name,User-ID:userId,My-Header-And-Tag-Name`<br>
+Accepts a map of case-insensitive header keys to tag names and automatically applies matching header values as tags on traces. Also accepts entries without a specified tag name that are automatically mapped to tags on the form `http.request.headers.*` and `http.response.headers.*` respectively.<br><br>
```

Comment:
should
> ... mapped to tags on the form ...

be
> ... mapped to tags of the form ...

also should we change the example to be a bit clearer, for example instead of
> `http.request.headers.*`

have
> `http.request.headers.<header-name>`